cmd/opm/alpha/generate/cmd.go

```diff
@@ -0,0 +1,104 @@
+package generate
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/internal/action"
+	"github.com/operator-framework/operator-registry/pkg/containertools"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{Use: "generate"}
+	cmd.AddCommand(
+		newDockerfileCmd(),
+	)
+	return cmd
+}
+
+func newDockerfileCmd() *cobra.Command {
+	var (
+		baseImage      string
+		extraLabelStrs []string
+	)
+	cmd := &cobra.Command{
+		Use:   "dockerfile <dcRootDir>",
+		Args:  cobra.ExactArgs(1),
+		Short: "Generate a Dockerfile for a declarative config index",
+		Long: `Generate a Dockerfile for a declarative config index.
+
+This command creates a Dockerfile in the same directory as the <dcRootDir>
+(named <dcDirName>.Dockerfile) that can be used to build the index. If a
+Dockerfile with the same name already exists, this command will fail.`,
+		RunE: func(_ *cobra.Command, args []string) error {
+			fromDir := args[0]
+
+			extraLabels, err := parseLabels(extraLabelStrs)
+			if err != nil {
+				return err
+			}
+
+			dir, indexName := filepath.Split(fromDir)
+			dockerfilePath := filepath.Join(dir, fmt.Sprintf("%s.Dockerfile", indexName))
+
+			if err := ensureNotExist(dockerfilePath); err != nil {
+				logrus.Fatal(err)
+			}
+
+			if s, err := os.Stat(fromDir); err != nil {
+				return err
+			} else if !s.IsDir() {
+				return fmt.Errorf("provided root path %q is not a directory")
```

Comment:
This appears to be missing the corresponding variable to that format specifier.

Reply:
Good catch. Thanks!